### PR TITLE
Each 176 Theme support added. A little refactored

### PR DESCRIPTION
**Pull request by ponjoru**
WARNING: if the project isn't running after merging add few strings into 'react-navigation.js:
get BottomTabBar() {
      return require('react-navigation-tabs').BottomTabBar;
  }

### CODE REVIEW

File: src/utils/constants.js [hunk 1]
```diff
@@ -1,14 +1,51 @@
...
+    HEADER_DARK: require('./../../assets/images/muz_header_dark.png'),
+    HEADER_LIGHT: require('./../../assets/images/muz_header_white.png'),
+};
+
+export const HeaderHeight = 40;
+export const TabLabelFontSize = 12;
+
 export const RESTART_ON_REMOUNT = '@@saga-injector/restart-on-remount';
 export const DAEMON = '@@saga-injector/daemon';
 export const ONCE_TILL_UNMOUNT = '@@saga-injector/once-till-unmount';
```
Comment (LanskovNV) [inline src/utils/constants.js]: end of line was forgotten

File: src/containers/Question/LocationQuestion.js [hunk 2]
```diff
@@ -1,20 +1,64 @@
...
+        justifyContent: 'center',
+        paddingTop: 25,
+        backgroundColor: '#ecf0f1',
+    },
+    paragraph: {
+        margin: 24,
+        fontSize: 18,
+        textAlign: 'center',
+    },
+});
```
Comment (LanskovNV) [inline src/containers/Question/LocationQuestion.js]: end of line was forgotten

File: src/components/MapStyles.js [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+    {
+        "featureType": "water",
+        "elementType": "labels.text.fill",
+        "stylers": [
+            {
+                "color": "#9e9e9e"
+            }
+        ]
+    }
+];
```
Comment (LanskovNV) [inline src/components/MapStyles.js]: end of line was forgotten

File: src/utils/constants.js (hunk 1 above)
Comment (ponjoru) [reply src/utils/constants.js]: @LanskovNV  done

File: src/containers/Question/LocationQuestion.js (hunk 2 above)
Comment (ponjoru) [reply src/containers/Question/LocationQuestion.js]: @LanskovNV done

File: src/components/MapStyles.js (hunk 3 above)
Comment (ponjoru) [reply src/components/MapStyles.js]: @LanskovNV done

**Pull request merged**